tests/dialects/test_pdl.py

```diff
@@ -1,16 +1,25 @@
+import pytest
 from xdsl.ir import Block
 
 import xdsl.dialects.pdl as pdl
-from xdsl.dialects.builtin import StringAttr
+from xdsl.dialects.builtin import ArrayAttr, IntegerAttr, StringAttr
+from xdsl.utils.exceptions import VerifyException
 
 type_type = pdl.TypeType()
 attribute_type = pdl.AttributeType()
+value_type = pdl.ValueType()
+operation_type = pdl.OperationType()
 
 
 def test_build_ops():
```

Comment:
I have to say that this test is becoming a bit too large
Shouldn't we separate each constructor to a single test? That should help debugging and make things cleaner.